apigw/src/shared/auth/customer-saml.ts

```diff
@@ -28,16 +33,20 @@ function createKeycloakSamlStrategy(
   })
   return new SamlStrategy(
     {
-      issuer: 'evaka-customer',
-      callbackUrl: samlConfig.callbackUrl,
-      entryPoint: samlConfig.entryPoint,
-      logoutUrl: samlConfig.entryPoint,
       acceptedClockSkewMs: -1,
+      cacheProvider: redisClient
+        ? redisCacheProvider(redisClient, { keyPrefix: 'suomifi-saml-resp:' })
```

Comment:
Citizen SAML should probably use a different prefix than actual suomi.fi SAML

Reply:
Goddamit :D